osv2/mget.go

```diff
@@ -94,14 +95,34 @@ func (m *MGetRequest) Do(ctx context.Context, client *opensearch.Client) (*MGetR
 	return resp, nil
 }
 
-// FromModelMGetRequest creates a new [mgetRequest] from the given [opensearchtools.MGetRequest].
-func FromModelMGetRequest(req *opensearchtools.MGetRequest) *MGetRequest {
+// FromDomainMGetRequest creates a new [mgetRequest] from the given [opensearchtools.MGetRequest].
+func FromDomainMGetRequest(req *opensearchtools.MGetRequest) *MGetRequest {
```

Comment:
So it doesn't slip through the cracks ...
https://github.com/CrowdStrike/opensearchtools/pull/13#discussion_r1117309719